### PR TITLE
PEP8Bear.py: Add asciinema 

### PR DESCRIPTION
**Pull request by MacBox7**
URL for asciinema has been added

Closes https://github.com/coala/coala-bears/issues/873

### CODE REVIEW

File: bears/python/PEP8Bear.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class PEP8Bear(LocalBear):
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/163081'
```
Comment (sangamcse) [inline bears/python/PEP8Bear.py]: LGTM
cc @Makman2

Comment (Makman2) [inline bears/python/PEP8Bear.py]: Looks good. But is it just me on my browser, or is the width too small of the asciinema? The text is cut-off for me :/

**Comment by MacBox7** [comment]
Why is the PR failing? When I am simply merging my branch with the master and pushing my changes?

**Comment by sangamcse** [comment]
Why are you merging your branch?

**Comment by MacBox7** [comment]
I am updating my branch with the coala/master then updating it with changes. Otherwise, git doesn't allow to push.

**Comment by sangamcse** [comment]
Use `git push -f origin PEP8BearDemo` then.
> I am updating my branch with the coala-bears/master then updating it with changes. Otherwise, git doesn't allow to push.

**Comment by MacBox7** [comment]
These commands closed this PR.

**Comment by sangamcse** [comment]
You have closed it accidentally. Reopen this using `git commit` and `git push -f origin PEP8BearDemo`.

**Comment by MacBox7** [comment]
Did as you suggested @sangamcse . Still failing.

**Comment by sangamcse** [comment]
Look at https://github.com/coala/coala-bears/pull/2281/commits. You only need 4c60b0238bb2df2ed934acc0199f63a6a61dd7ec.

**Comment by MacBox7** [comment]
That was a previous commit in which the text was not clear as said by Makman2. So I created a new asciinema and now I want to update the previous link.

**Comment by MacBox7** [comment]
Should I close this PR and create a new PR ?

**Comment by sangamcse** [comment]
That was previous commit that I can understand but then you have to use `git commit --amend` instead of `git commit`. Don't close the PR. Change your commit HEAD to 4c60b0238bb2df2ed934acc0199f63a6a61dd7ec using `git reset --hard HEAD@{2}` then change the requested changes and then `git commit --amend` and `git push -f origin PEP8BearDemo`.

**Comment by sangamcse** [comment]
Now your commit is good. If you are doing any changes in files, then after that use `git add thatfile`, `git commit --amend` then `git push -f origin yourbranch`. I hope you got it clearly.

**Comment by MacBox7** [comment]
Yup, thanks for the help. Is there any way to know why the checks are failing? The details don't seem to redirect to error log.

**Comment by sangamcse** [comment]
I don't think it will fail now. It was failing because you pulled upstream and pushed that in this branch.

**Comment by MacBox7** [comment]
Could the ci be rebuilt?

**Comment by Makman2** [comment]
I still see that the lines are cut-off because the "video"'s width is too small.

**Comment by MacBox7** [comment]
@Makman2 Shouldn't be a problem now. Updated a fresh one.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2281/commits/45de414b58f87a247ddab69ea569f81aaaae86a5

**Comment by jayvdb** [comment]
ack 45de414

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**